### PR TITLE
Do not use std::bind2nd, which is removed in C++17.

### PR DESCRIPTION
**Pull request by atgeirr**

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

File: opm/core/transport/implicit/JacobianSystem.hpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ namespace Opm {
                 typedef typename BaseVec::value_type VT;
                 ::std::transform(x.begin(), x.end(),
                                  y.begin(),
-                                 ::std::bind2nd(::std::multiplies<VT>(), a));
+                                 [&a](auto x) { return a * x; });
```
Comment (bska) [inline opm/core/transport/implicit/JacobianSystem.hpp]: This will render the `VT` type alias unused.  You should remove that too.

Comment (atgeirr) [reply opm/core/transport/implicit/JacobianSystem.hpp]: > This will render the `VT` type alias unused. You should remove that too.

Update force-pushed.

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
Since the review comment has been adressed, the maintainer has approved, and jenkins is green I will self-merge this.

**Pull request merged**